config/_default/params.toml

```diff
@@ -119,7 +119,7 @@ contact_links = [
 ############################
 
 # Default image for social sharing and search engines. Place image in `static/img/` folder and specify image name here.
-sharing_image = ""
+sharing_image = "logo-metadocencia.png"
```

Comment:
otherwise in the partial avatar is used but avatar isn't a filepath, it's a map.